go.mod

```diff
@@ -3,26 +3,17 @@ module github.com/api7/ingress-controller
 go 1.13
 
 require (
-	github.com/coreos/etcd v3.3.18+incompatible
-	github.com/coreos/go-semver v0.3.0 // indirect
-	github.com/coreos/go-systemd v0.0.0-20180511133405-39ca1b05acc7 // indirect
-	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
 	github.com/gogo/protobuf v1.3.1 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/google/uuid v1.1.1 // indirect
 	github.com/gxthrj/apisix-ingress-types v0.1.2
 	github.com/gxthrj/apisix-types v0.1.0
 	github.com/gxthrj/seven v0.1.9
 	github.com/julienschmidt/httprouter v1.3.0
-	github.com/pkg/errors v0.8.1
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
```

Comment:
How about use `mitchellh/colorstring` directly, `k0kubun/colorstring` is forked from it.